.github/dependabot.yml

```diff
@@ -20,16 +20,20 @@ updates:
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
-      interval: "daily"
+      interval: weekly
+      time: '15:00'
```

Comment:
Do we need to set a timezone? The [docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduletimezone) seem to not be 100% accurate, but they try to indicate that it may be somewhat random?

Reply:
I was hoping it would default to UTC, but I can make that explicit...